Task2/Project/Test/Test.csproj

```diff
@@ -35,10 +35,13 @@
     <WarningLevel>4</WarningLevel>
   </PropertyGroup>
   <ItemGroup>
+    <Reference Include="PresentationCore" />

```

Comment:
for what that reference stand for?

Reply:
I dont remember but VS added it, not me

Reply:
Can you remove it?

Reply:
> 
> 
> Can you remove it?

It gives you System.Windows.Media.Color; so it should be present

Reply:
oh, so that what it does :sweat_smile:, then we should definately keep it